examples/async-streaming/client/phone_pb.bal

```diff
@@ -0,0 +1,185 @@
+    isolated remote function receiveStreamCallResponse() returns StreamCallResponse|grpc:Error? {
+        var response = check self.sClient->receive();
+        if response is () {
+            return response;
+        } else {
+            [anydata, map<string|string[]>] [payload, _] = response;
+            return <StreamCallResponse>payload;
+        }
+    }
+
+    isolated remote function receiveContextStreamCallResponse() returns ContextStreamCallResponse|grpc:Error? {
+        var response = check self.sClient->receive();
+        if response is () {
+            return response;
+        } else {
+            [anydata, map<string|string[]>] [payload, headers] = response;
+            return {content: <StreamCallResponse>payload, headers: headers};
+        }
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.sClient->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.sClient->complete();
+    }
+}
+
+public client class PhoneStreamCallResponseCaller {
+    private grpc:Caller caller;
+
+    public isolated function init(grpc:Caller caller) {
+        self.caller = caller;
+    }
+
+    public isolated function getId() returns int {
+        return self.caller.getId();
+    }
+
+    isolated remote function sendStreamCallResponse(StreamCallResponse response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendContextStreamCallResponse(ContextStreamCallResponse response) returns grpc:Error? {
+        return self.caller->send(response);
+    }
+
+    isolated remote function sendError(grpc:Error response) returns grpc:Error? {
+        return self.caller->sendError(response);
+    }
+
+    isolated remote function complete() returns grpc:Error? {
+        return self.caller->complete();
+    }
+
+    public isolated function isCancelled() returns boolean {
+        return self.caller.isCancelled();
+    }
+}
+
+public type ContextStreamCallResponseStream record {|
+    stream<StreamCallResponse, error?> content;
+    map<string|string[]> headers;
+|};
+
+public type ContextStreamCallRequestStream record {|
+    stream<StreamCallRequest, error?> content;
+    map<string|string[]> headers;
+|};
+
+public type ContextStreamCallResponse record {|
+    StreamCallResponse content;
+    map<string|string[]> headers;
+|};
+
+public type ContextStreamCallRequest record {|
+    StreamCallRequest content;
+    map<string|string[]> headers;
+|};
+
+public type StreamCallResponse record {|
+    CallInfo call_info?;
+    CallState call_state?;
+|};
+
+isolated function isValidStreamcallresponse(StreamCallResponse r) returns boolean {
+    int stream_call_responseCount = 0;
+    if !(r?.call_info is ()) {
+        stream_call_responseCount += 1;
+    }
+    if !(r?.call_state is ()) {
+        stream_call_responseCount += 1;
+    }
+    if (stream_call_responseCount > 1) {
```

Comment:
```suggestion
    if stream_call_responseCount > 1 {
```

Reply:
This is actually the stub file, we need to change the code there.